source/SIL.AppBuilder.Portal.Frontend/src/data/containers/sorting.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
```

Comment:
Not sure we need to redo our own component. Semantic table can do sorting.

https://react.semantic-ui.com/collections/table#variations-sortable

Reply:
I need to hook in to orbit, so I'll be ignoring semantic's 'features' :)